### PR TITLE
Add ability to redirect bash for run -i

### PR DESCRIPTION
**Pull request by baude**
Signed-off-by: baude <bbaude@redhat.com>

### CODE REVIEW

File: pkg/varlinkapi/virtwriter/virtwriter.go [hunk 1]
```diff
@@ -161,7 +167,16 @@ func Reader(r *bufio.Reader, output, errput, input io.Writer, resize chan remote
 				execEcChan <- int(ecInt)
 			}
 			return nil
-
+		case HangUpFromClient:
+			// This sleep allows the pipes to flush themselves before tearing everything down.
```
Comment (rhatdan) [inline pkg/varlinkapi/virtwriter/virtwriter.go]: Are you doing it?  Seems like you have commented it out?

**Comment by rhatdan** [comment]
Since you are claiming this will fix and error, can you add a test for that error?

File: pkg/varlinkapi/virtwriter/virtwriter.go (hunk 1 above)
Comment (baude) [reply pkg/varlinkapi/virtwriter/virtwriter.go]: fixed

**Comment by baude** [comment]
> Since you are claiming this will fix and error, can you add a test for that error?

i believe this is caught in system test.

**Comment by rhatdan** [comment]
LGTM
@jwhonce @mheon @giuseppe @vrothberg @TomSweeneyRedHat @QiWang19 PTAL

**Pull request merged**